views/js/uiForm.js

```diff
@@ -506,23 +506,25 @@
                 return new Promise((resolve, reject) => {
                     if (!dependencies.length) {
                         return regularConfirmantion() ? resolve() : reject();
-                    } else {
-                        const name = $groupNode.find('.property-heading-label')[0].innerText;
-                        const dependantPropName = dependencies.reduce((prev, next, index) => {
-                            const delimiter = index === dependencies.length - 1 ? '' : ', '
-                            return prev + `${next.label}${delimiter}`;
-                        }, '');
-
-                        confirmDialog(
-                            `<b>${name}</b>
-                            ${__('currently has a dependency established with ')}
-                            <b>${dependantPropName}</b>.
-                            ${__('Deleting this property will also remove the dependency')}.
-                            <br><br> ${__('Are you wish to delete it')}?`,
-                            resolve,
-                            reject
-                        );
                     }
+
+                    const name = $groupNode.find('.property-heading-label')[0].innerText;
+                    const dependantPropName = dependencies.reduce((prev, next, index) => {
+                        const delimiter = index === dependencies.length - 1 ? '' : ', '
+                        return prev + `${next.label}${delimiter}`;
+                    }, '');
+
+                    let message = `<b>${name}</b>
```

Comment:
nitpick: I think this can be a `const` as well

```suggestion
                    const message = `<b>${name}</b>
```

Reply:
Good catch!